gaia-switch.js

```diff
@@ -44,11 +44,31 @@ module.exports = component.register('gaia-switch', {
     this.disabled = this.getAttribute('disabled');
     this.checked = this.getAttribute('checked');
 
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
```

Comment:
Let me try again if below comment is more clear: :sweat_smile:  The idea is to: 

`process everything not effect user interaction immediately when component is created, so the UI can load faster`. 

Set element attributes is just one of these cases.